### PR TITLE
Fixed Import Image Text Overflow 

### LINKED ISSUE #693: Fix Import Image Text Overflow
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### What is the problem
The file input text for import-image module is overflowing whereas for the load-image(first step) it is not. Fix that by changing an id in css to a class.

Import Image:

screenshot from 2019-01-21 11-56-10

Expected:

screenshot from 2019-01-21 11-56-59

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [demo.css](image-sequencer/examples/demo.css) in the `image-sequencer` repository (press the little pen Icon) and edit the line as shown below. 

Change

https://github.com/publiclab/image-sequencer/blob/17d79c4b44c5e5b8f7f8b4574a783d5b87cdf871/examples/demo.css#L49-L51

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff css
- #dropzone {
+ .dropzone {
  padding: 30px;
```
Change

https://github.com/publiclab/image-sequencer/blob/17d79c4b44c5e5b8f7f8b4574a783d5b87cdf871/examples/demo.css#L62-L64

```diff
- #dropzone input {
+ .dropzone input {
  max-width: 100%;
}
```

- [ ] 📝 **Update** the file [index.html](image-sequencer/examples/index.html) in the `image-sequencer` repository (press the little pen Icon) and edit the line as shown below. 

Change

https://github.com/publiclab/image-sequencer/blob/17d79c4b44c5e5b8f7f8b4574a783d5b87cdf871/examples/index.html#L65-L67

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
-  <div id="dropzone">
+  <div id="dropzone" class="dropzone">
      <p>
         <i>Select or drag in an image to start!</i>
```

- [ ] 📝 **Update** the file [Ui.js](image-sequencer/src/modules/ImportImage/Ui.js) in the `image-sequencer` repository (press the little pen Icon) and edit the line as shown below. 

Change

https://github.com/publiclab/image-sequencer/blob/17d79c4b44c5e5b8f7f8b4574a783d5b87cdf871/src/modules/ImportImage/Ui.js#L10-L11

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
    var dropZone ='\
-    <div style="padding: 30px;margin: 10px 20% 30px;border: 4px dashed #ccc;border-radius: 8px;text-align: center;color: #444;" id="' + dropzoneId + '">\
+    <div class="dropzone" style="padding: 30px;margin: 10px 20% 30px;border: 4px dashed #ccc;border-radius: 8px;text-align: center;color: #444;" id="' + dropzoneId + '">\
```

[See this page](https://publiclab.github.io/community-toolbox/#r=all) for some help in taking your first steps!

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

### PR DESCRIPTION
**Pull request by najMaseu**
Fixes #693
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by HarshKhandeparkar** [comment]
@najMaseu welcome and thanks for your PR!
I have a few suggestions:
1. Please change the name of the PR to be more descriptive.
2. You can check all of the checkboxes to make it easier for reviewers to review.
3. You can mention @publiclab/is-reviewers to ask for a review in a PR.
4. In the `demo.css` file, one change is still remaining. Please commit that.

Thanks!

File: examples/demo.css [hunk 1]
```diff
@@ -46,7 +46,7 @@ h1 {
   padding: 0px 0px;
 }
 
-#dropzone {
```
Comment (HarshKhandeparkar) [inline examples/demo.css]: Please make these changes on line 62 

```diff
- #dropzone input {
+ .dropzone input {
  max-width: 100%;
}
```

**Comment by najMaseu** [comment]
I've made requested changes. May I ask for a review? @publiclab/is-reviewer

**Comment by rexagod** [comment]
@publiclab/is-reviewers

**Comment by najMaseu** [comment]
Thank you guys!

**Comment by HarshKhandeparkar** [comment]
:+1:. @jywarren this is ready for a final review

**Comment by HarshKhandeparkar** [comment]
@jywarren sorry for pinging. Wrong reply in the wrong place. @najMaseu sorry

**Pull request merged**